src/app/features/ledger/hooks/use-ledger-navigate.ts

```diff
@@ -145,10 +145,7 @@ export function useLedgerNavigate() {
       },
 
       cancelLedgerAction() {
-        // Use baseUrl to determine where to go on close
-        const baseUrl = `/${location.pathname.split('/')[1]}`;
-
-        return navigate(baseUrl, {
+        return navigate('..', {
```

Comment:
Did you track back to [the PR where I changed it](https://github.com/leather-io/extension/pull/5353/files#r1594993336)? I think changing it back to '..' will re-introduce [this bug](https://github.com/leather-io/extension/issues/5143).

If you're confident this will work fair enough. I think the problem you hit is a new bug and could be solved by making the `baseUrl` find the route you are on more accurately